etc/profile-m-z/okular.profile

```diff
@@ -58,7 +58,9 @@ protocol unix
 seccomp
 tracelog
 
-private-bin kbuildsycoca4,kdeinit4,lpr,okular,unar,unrar
+private-bin kbuildsycoca4,kdeinit4,lpr,okular,unar,unrar,ps2pdf
+# ps2pdf - enables print to pdf functionality
```

Comment:
Remove both this and the following line. We have the outdated info in https://github.com/netblue30/firejail/discussions/6402 if we want to change that in the future.

Reply:
k. i usually comment all the things I allow on my app armor profiles. was just going along here too.